src/main/java/tc/oc/pgm/rotation/FixedPGMMapOrder.java

```diff
@@ -134,7 +128,9 @@ PGMMap getMapInPosition(int position) {
                   + maps.size()
                   + " has been issued. Returning map in position 0 instead.");
       return getMapInPosition(0);
-    } else return maps.get(position);
+    }
+
+    return maps.get(position % maps.size());
```

Comment:
```suggestion
    return maps.get(position);
```
Since the input is restricted, the % for normalization isn't needed

Reply:
Fixed! https://github.com/Electroid/PGM/pull/120/commits/ee1be6fa4580db48098150a6d586a056038b5afd